notify_login_email_changed/views/email_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="email_template_notify_email_change" model="mail.template">
+        <field name="name">Email Changed</field>
+        <field name="email_from">${ctx['new_email']|safe}</field>
```

Comment:
We dont need to declare `email_from` since this email should be sent from the system.

Reply:
I removed it